plugins/modules/azure_rm_monitoractivitylogalerts_info.py

```diff
@@ -0,0 +1,240 @@
+            "location": "global",
+            "name": "activity_log_alert_name",
+            "scopes": [
+                "/subscriptions/xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/resource_group_name"
+            ],
+            "tags": {},
+            "type": "Microsoft.Insights/ActivityLogAlerts"
+        }
+    ]
+'''
+
+from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common import AzureRMModuleBase
+
+try:
+    from azure.core.exceptions import ResourceNotFoundError
+
+except ImportError:
+    # This is handled in azure_rm_common
+    pass
+
+AZURE_OBJECT_CLASS = 'activitylogalerts'
+
+
+class AzureRMactivitylogalertsInfo(AzureRMModuleBase):
+    """Information class for an Azure RM Activity log alerts"""
+
+    def __init__(self):
+        self.module_arg_spec = dict(
+            name=dict(type='str', required=False),
+            resource_group=dict(type='str', required=False),
+        )
+
+        self.name = None
+        self.resource_group = None
+        self.log_path = None
+        self.log_mode = None
+
+        self.results = dict(
+            changed=False,
+            activitylogalerts=[]
+        )
+
+        super(AzureRMactivitylogalertsInfo, self).__init__(derived_arg_spec=self.module_arg_spec,
+                                                           supports_check_mode=True,
+                                                           supports_tags=False,
+                                                           facts_module=True)
+
+    def exec_module(self, **kwargs):
+        """Main module execution method"""
+
+        for key in self.module_arg_spec:
+            setattr(self, key, kwargs[key])
+
+        if self.name:
+            result = self.get_activity_log_alert()
+        else:
+            result = self.list_activity_log_alerts()
+
+        self.results['activitylogalerts'] = result
+
+        return self.results
+
+    def get_activity_log_alert(self):
+        '''
+        Gets the properties of the specified activity log alert.
+
+        :return: List of Activity log alerts
+        '''
+        self.log("Checking if activity log alert {0} in resource group {1} is present".format(self.name,
+                                                                                              self.resource_group))
+
+        result = []
+        activity_log_alert = None
+
+        try:
+            activity_log_alert = self.monitor_management_client_activity_log_alerts.activity_log_alerts.get(activity_log_alert_name=self.name,
+                                                                                                            resource_group_name=self.resource_group)
+        except ResourceNotFoundError as ex:
+            self.log("Could not find activity log alert {0} in resource group {1}".format(self.name, self.resource_group))
+            self.fail("activity log alert {0} not found in resource group {1}".format(self.name, self.resource_group))
```

Comment:
When it cannot be obtained, it should return to null instead of reporting an error.
```suggestion
           return []
```